contracts/src/v0.1/Prepayment.sol

```diff
@@ -393,6 +406,7 @@ contract Prepayment is Ownable, PrepaymentInterface, TypeAndVersionInterface {
             }
         }
         s_coordinators.push(CoordinatorBaseInterface(coordinator));
+        sIsCoordinators[coordinator] = true;
```

Comment:
Since we introduced the mapping which holds all active coordinators, we can simplify this function a lot. It could look as in the listing below.

```Solidity
function addCoordinator(address coordinator) public onlyOwner {
    if (sIsCoordinators[coordinator]) {
        revert CoordinatorExists();
    }
    s_coordinators.push(CoordinatorBaseInterface(coordinator));
    sIsCoordinators[coordinator] = true;
}
```